src/main/java/edu/harvard/iq/dataverse/AuxiliaryFile.java

```diff
@@ -44,6 +48,29 @@
     
     private String checksum;
 
+    /**
+     * A way of grouping similar auxiliary files together. The type could be
+     * "DP" for "Differentially Private Statistics", for example.
+     */
+    @Column(nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Type type;
+
+    public enum Type {
```

Comment:
rather than an enum, can we make this a String? the way to handle the friendly text (for now) would be to just return the value for OTHER for anything other than DP. (eventually this would have to be more robust and first check to see if a bundle value exists before defaulting to other.

Reply:
Done in 9e5f39089.